modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,119 @@
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-13-0_{context}"]
+== New features
+
+* With this update, you can configure Argo CD to disable the automatic scraping of metrics for your instance. Before this update, there was no option to disable this feature, resulting in excessive storage usage when multiple Argo CD instances were available on the cluster. link:https://issues.redhat.com/browse/GITOPS-4519[GITOPS-4519]
++
+[NOTE]
+====
+In {gitops-shortname}, the scraping of metrics is enabled by default for all Argo CD instances.
+====
++
+To disable metrics for your Argo CD instance, edit the `spec.monitoring.disableMetrics` field in the Argo CD custom resource to `false`.
++
+.Example 
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: Argo CD
+metadata:
+ name: Argo CD-sample
+ namespace: default
+spec:
+ monitoring:
+   disableMetrics: true
+----
++
+Use the following command to scrape metrics in the Argo CD CLI:
++
+[source,terminal]
+----
+$ oc patch Argo CD Argo CD-sample -n default --type='json' -p='[{"op": "replace", "path": "/spec/monitoring/disableMetrics", "value": true}]'
+----
+
+* With this update, the Argo Rollouts functionality in {gitops-title} is promoted from Technology Preview (TP) to General Availability (GA). Use this feature to perform the following actions: link:https://issues.redhat.com/browse/GITOPS-3848[GITOPS-3848]
+
+** Run cluster-scoped Argo Rollouts instead of namespace-scoped
+** Route traffic by using Openshift Routes for canary application deployment 
+** Configure Argo Rollouts CLI
+
+* With this update, you can perform canary-based Argo Rollouts deployments by using OpenShift Routes-based traffic management. Argo Rollouts supports traffic splitting by modifying the percentage of traffic that reaches the canary or stable application version during a deployment. link:https://issues.redhat.com/browse/GITOPS-2400[GITOPS-2400]
+
+* With this update, the Argo CD Applications in any namespace feature is promoted from Technology Preview (TP) to General Availability (GA). link:https://issues.redhat.com/browse/GITOPS-3796[GITOPS-3796]
+
+* With this update, the following new commands are introduced for the Apps in any namespace feature:
+** `argocd proj add-source-namespace` - Use this command to add source namespaces to `AppProject.Spec.SourceNamespaces`, for whitelisting specific namespaces for application creation.
+** `argocd proj delete-source-namespace` - Use this command to remove source namespaces from `AppProject.Spec.SourceNamespaces`, for blacklisting specific namespaces. link:https://issues.redhat.com/browse/GITOPS-4045[GITOPS-4045]
```

Comment:
small nit- I think you're missing maybe an "and" before "for" after the comma? It doesn't read clearly to me

Reply:
Yes, the addition of the and does make the sentence more readable. Thanks for catching this nit. Fixed.